src/dashboard/Dashboard.spec.js

```diff
@@ -9,6 +9,8 @@ describe("control button click functions", () => {
   const dash = render(<Dashboard />);
   const lockButton = dash.getByTestId("lockButton");
```

Comment:
I love that you created these variables on top of the file instead of calling it for every test!! Cleans up your code much more.